src/api/.rubocop.yml

```diff
@@ -141,6 +141,9 @@ Style/SpaceInsideBrackets:
 Style/TernaryParentheses:
   Enabled: true
 
+# Checks for receiver.length == 0 predicates and the negated versions receiver.length > 0 and receiver.length != 0.
```

Comment:
I know that this is what the Rubocop documentation says, but it is not accurate as it also checks for things like `.length < 1` and it also works for `size`.